src/status_im/hardwallet/core.cljs

```diff
@@ -257,20 +265,32 @@
        :hardwallet/get-keys {:pairing pairing
                              :pin     pin}})))
 
+(defn- get-account-for-login
+  [db key-uid instance-uid auto-login?]
+  (if auto-login?
+    (or
+     (find-account-by-keycard-key-uid db key-uid)
+     ; NOTE: temporary fix to allow accounts without key-uid to login using instance-uid
```

Comment:
why is it a temporary fix? what is need to make it permanent?

Reply:
@yenda To remove this workaround we need all accounts created with keycard before this fix to have key-uid. 
I've added `hardwallet/save-key-uid-to-account` which is called when keycard account does login to ensure old accounts will eventually have key-uid stored.

Reply:
@dmitryn it is not possible to currently create an account with a keycard in the releases right? So the intention is to remove this for the next release or something?

Reply:
@yenda yes, it should be removed before official release. Now apk with keycard functionality was only released unofficially in #status-keycard channel. So this fix is only for some of the very pro-active users who might actually started playing with keycard before the release.

Reply:
could you add this information in the docstring just in case it gets forgotten so that someone reading that code knows what'sup

Reply:
added